docs/content/latest/deploy/kubernetes/single-zone/oss/helm-chart.md

```diff
@@ -143,6 +143,16 @@ resource.tserver.requests.cpu=0.5,resource.tserver.requests.memory=0.5Gi,\
 enableLoadBalancer=False --namespace yb-demo
 ```
 
+{{< note title="Important" >}}
+
+On some environments like macOS, minikube may run inside a virtual machine. Make sure to configure the vm with at least 4 cpus and 8GB memory so 
+the cluster has room to startup. Example config in macOS:
+```
+VBoxManage modifyvm "minikube" --cpus 4 --memory 8192
```

Comment:
it would be better to just specify these with minikube start like `minikube start --cpus 4 --memory 5120` instead of through VBoxManage. Technically, minikube can use drivers other than virtualbox.

Also 8 GB is much higher than 6 * 0.5G, I would suggest something like 5G instead.

Reply:
Commited changes. @iSignal please recheck